test/fixtures/dtl/for-key.dtl

```diff
@@ -1,3 +1,3 @@
-{% for key, items_i in items %}
-  {{ key }}: {{ items_i }}
+{% for items_i in items %}
```

Comment:
When using `@key`, we can assume it's for object-iteration, so in that case we should switch to the same output as using `|value key|`.